tasks-repository.go

```diff
@@ -1,65 +1,55 @@
 package main
 
 import (
-	"database/sql"
 	"fmt"
+	"strconv"
 )
 
-// Returns all tasks in SQL database
-func getTasks_sql() ([]Task, error) {
+// Returns all tasks in SQL database that match input filter
+func getTasksByFilter_sql(searchedID string, searchedTitle string, searchedComplete string) ([]Task, error) {
```

Comment:
Do not specify the language you are using to execute this function. Remove the _sql part from the name. This function should be able to be refactored to use a different method and still be valid